tests/e2e/pom/project-settings-model.ts

```diff
@@ -0,0 +1,23 @@
+import type { Page } from "@playwright/test";
+
+export class DevProjectSettingsPage {
+    readonly page: Page;
+    readonly projectNameInput;
+    readonly renameButton;
```

Comment:
Annotate the type so that it is clear what the types are.
```suggestion
    readonly projectNameInput: Locator;
    readonly renameButton: Locator;
```